hypha/apply/projects/models/project.py

```diff
@@ -547,6 +567,11 @@ class ProjectSettings(BaseSiteSetting, ClusterableModel):
                 "to move all internal approval projects back to the 'Draft' stage with all approvals removed."
             ),
         ),
+        InlinePanel(
+            "reminder_frequencies",
+            label=_("Report reminder frequency"),
+            heading=_("Report reminder frequency"),
```

Comment:
Can we add a help text here? something to explain that these settings are used when running `notify_report_due.py` as a cronjob. Please improve my draft below.

```
help_text=_("Set up a cronjob to run `notify_report_due.py`. That script will use the reminder settings below.")
```

Reply:
I should look a little more closely at the exact command I don't see `notify_report_due.py` in example cron jobs currently.